### PR TITLE
build a new extension for simpler tests

### PR DESCRIPTION
**Pull request by awestover**
## Problem
Remote Extension tests are non-deterministic. 
## Summary of changes
use simpler extensions for better tests

### CODE REVIEW

File: Dockerfile.compute-node [hunk 1]
```diff
@@ -664,6 +664,29 @@ RUN wget https://github.com/kelvich/pg_tiktoken/archive/801f84f08c6881c8aa30f405
...
+    mkdir weighted_mean-src && cd weighted_mean-src && tar xvzf ../weighted_mean.tar.gz --strip-components=1 -C . && \
+    find /usr/local/pgsql -type f | sed 's|^/usr/local/pgsql/||' > /before.txt &&\
+    make -j $(getconf _NPROCESSORS_ONLN) install PG_CONFIG=/usr/local/pgsql/bin/pg_config && \
+    echo 'trusted = true' >> /usr/local/pgsql/share/extension/weighted_mean.control && \
+    find /usr/local/pgsql -type f | sed 's|^/usr/local/pgsql/||' > /after.txt &&\
+    mkdir -p /extensions/weighted_mean && cp /usr/local/pgsql/share/extension/weighted_mean.control /extensions/weighted_mean && \
+    sort -o /before.txt /before.txt && sort -o /after.txt /after.txt && \
+    comm -13 /before.txt /after.txt | tar --directory=/usr/local/pgsql --zstd -cf /extensions/weighted_mean.tar.zst -T -
+
+
```
Comment (bayandin) [inline Dockerfile.compute-node:689]: We don't need to build it for computes

File: test_runner/regress/data/extension_test/5670669815/v14/extensions/weighted_mean.tar.zst
Comment (bayandin) [inline test_runner/regress/data/extension_test/5670669815/v14/extensions/weighted_mean.tar.zst:1]: We need to add the source code of the extension to the repo, maybe as a submodule (mention it in NOTICE file) and build it as a part of make `postgres-%` job (create tar.zst archive), or create a separate job if you want.
And then use it in tests

File: Dockerfile.compute-node (hunk 1 above)
Comment (awestover) [reply Dockerfile.compute-node:689]: ok i guess not

Comment (awestover) [reply Dockerfile.compute-node:689]: makes sense

File: test_runner/regress/data/extension_test/5670669815/v14/extensions/weighted_mean.tar.zst
Comment (awestover) [reply test_runner/regress/data/extension_test/5670669815/v14/extensions/weighted_mean.tar.zst:1]: good call

Comment (awestover) [reply test_runner/regress/data/extension_test/5670669815/v14/extensions/weighted_mean.tar.zst:1]: ill find a notice / license

**Pull request closed without merging**